sql/mozfun/norm/product_info/README.md

```diff
@@ -1,20 +1,20 @@
-Focus            | iOS           | Focus iOS        | Firefox Focus for iOS       | true  | true
-Klar             | Android       | Klar Android     | Firefox Klar for Android    | false | false
-Klar             | iOS           | Klar iOS         | Firefox Klar for iOS        | false | false
-Lockbox          | Android       | Lockwise Android | Lockwise for Android        | true  | true
-Lockbox          | iOS           | Lockwise iOS     | Lockwise for iOS            | true  | true
-FirefoxReality*  | Android       | Firefox Reality  | Firefox Reality             | false | false
+app_name         | normalized_os | looker_app_name  | product          | canonical_app_name              | 2019  | 2020  | 2021
+---------------- | ------------- | ---------------- | ---------------- | ------------------------------- | ----- | ----  | ----
+Firefox          | *             | firefox_desktop  | Firefox          | Firefox for Desktop             | true  | true  | true
+Fenix            | Android       | fenix            | Fenix            | Firefox for Android (Fenix)     | true  | true  | true
+Fennec           | Android       | fennec           | Fennec           | Firefox for Android (Fennec)    | true  | true  | true
+Firefox Preview  | Android       | firefox_preview  | Firefox Preview  | Firefox Preview for Android     | true  | true  | true
+Fennec           | iOS           | firefox_ios      | Firefox iOS      | Firefox for iOS                 | true  | true  | true
+FirefoxForFireTV | Android       | firefox_fire_tv  | Firefox Fire TV  | Firefox for Fire TV             | false | false | false
+FirefoxConnect   | Android       | firefox_connect  | Firefox Echo     | Firefox for Echo Show           | true  | true  | true
+Zerda            | Android       | firefox_lite     | Firefox Lite     | Firefox Lite                    | true  | true  | true
+Zerda_cn         | Android       | firefox_lite_cn  | Firefox Lite CN  | Firefox Lite (China)            | false | false | false
+Focus            | Android       | focus_android    | Focus Android    | Firefox Focus for Android       | true  | true  | true
+Focus            | iOS           | focus_ios        | Focus iOS        | Firefox Focus for iOS           | true  | true  | true
+Klar             | Android       | klar_android     | Klar Android     | Firefox Klar for Android        | false | false | false
+Klar             | iOS           | klar_ios         | Klar iOS         | Firefox Klar for iOS            | false | false | false
+Lockbox          | Android       | lockwise_android | Lockwise Android | Lockwise for Android            | true  | true  | true
+Lockbox          | iOS           | lockwise_ios     | Lockwise iOS     | Lockwise for iOS                | true  | true  | true
+FirefoxReality*  | Android       | firefox_reality  | Firefox Reality  | Firefox Reality                 | false | false | false
```

Comment:
Added @gkabbz for review as I understand he's working on KPI forecasting for mobile, and he reviewed the initial implementation of this function. I'm mostly looking for you to validate the additions to this table. The rest of the PR is code changes generated from it, plus tests.